tests/testthat/test-load-fbref.R

```diff
@@ -0,0 +1,32 @@
+context("Testing FBref Load data functions")
+
+
```

Comment:
how about a test to check for bad params?

Something like

```r
test_df <- load_match_results(country = "foo", gender = "M", season_end_year = 2022, tier="1st")
expect_type(test_df, "list") # 0-row df
expect_true(nrow(test_df) == 0)

```

Reply:
You probably don't need to do that for every `load_` function, but having at least one test would be nice

Reply:
I had this in a previous commit but was causing tests to fail for some reason... Unless I stuffed up by accidentally having `expect_false(nrow(test_df) == 0)`.

I'll implement it again